dbt/dbt_project.yml

```diff
@@ -66,8 +66,6 @@ models:
       +schema: proximity
     reporting:
       +schema: reporting
-    rpie:
-      +schema: rpie
```

Comment:
The presence of this config was throwing sporadic errors given that `rpie` models were all removed in https://github.com/ccao-data/data-architecture/pull/455.